### PR TITLE
Update Dependencies + TabList new style

### PR DESCRIPTION
**Pull request by hugomarquesdev**

### CODE REVIEW

File: src/components/Icon.tsx [hunk 1]
```diff
@@ -14,15 +14,15 @@ const renderPaths = (icon: { paths: any }) =>
...
+    onClick?: () => void;
 } & GeneratedPropTypes;
 
 const IconSvg = styled.svg<any>`
     fill: currentColor;
     flex-shrink: 0;
     vertical-align: middle;
 
-    ${({ size }) => !!size && setSize(size)};
+    ${({ size }: any) => !!size && setSize(size)};
```
Comment (obernardovieira) [inline src/components/Icon.tsx:25]: can this be
```suggestion
    ${({ size }: { size: SizeType }) => !!size && setSize(size)};
```
?

File: types/components/Accordion.d.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { GeneratedPropTypes } from '../types';
+import React from 'react';
+export declare type AccordionItemProps = {
+    children: any;
```
Comment (obernardovieira) [inline types/components/Accordion.d.ts:4]: using the same as above
```suggestion
    children: React.ReactNode;
```

File: types/components/Alert.d.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { BoolPropsFromArray, GeneratedPropTypes } from '../types';
+import { stateTypes } from '../helpers/applyStateColor';
+import React from 'react';
+export declare type AlertProps = {
+    button?: any;
+    icon: string;
+    title?: string | React.ReactNode;
+    message?: string | React.ReactNode;
+    handleClose?: any;
```
Comment (obernardovieira) [inline types/components/Alert.d.ts:9]: can this be something like
```suggestion
    handleClose?: () => void;
```
?

File: types/components/Alert.d.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { BoolPropsFromArray, GeneratedPropTypes } from '../types';
+import { stateTypes } from '../helpers/applyStateColor';
+import React from 'react';
+export declare type AlertProps = {
+    button?: any;
```
Comment (obernardovieira) [inline types/components/Alert.d.ts:5]: is it component or string?

File: types/components/Button.d.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+};
+declare type ButtonBaseProps = {
+    as?: any;
+    className?: string;
+    children?: any | any[];
+    disabled?: boolean;
+    href?: string;
+    icon?: string;
+    fluid?: Breakpoint | boolean;
+    onClick?: any;
```
Comment (obernardovieira) [inline types/components/Button.d.ts:24]: can this be something like
```suggestion
    onClick?: () => void;
```
?

File: types/components/Input.d.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+            disabled?: boolean | undefined;
+            hint?: string | undefined;
+            icon?: string | undefined;
+            onBlur?: Function | undefined;
+            onFocus?: Function | undefined;
+            prefix?: string | undefined;
+            rows?: string | number | undefined;
+            suffix?: string | undefined;
+            withError?: boolean | undefined;
+            wrapperProps?: GeneratedPropTypes | undefined;
```
Comment (obernardovieira) [inline types/components/Input.d.ts:33]: it's undefined already
```suggestion
            disabled?: boolean;
            hint?: string;
            icon?: string;
            onBlur?: Function;
            onFocus?: Function;
            prefix?: string;
            rows?: string | number;
            suffix?: string;
            withError?: boolean;
            wrapperProps?: GeneratedPropTypes;
```

File: package.json [hunk 7]
```diff
@@ -1,81 +1,88 @@
...
+        "@types/react": "17.0.39",
+        "@types/react-dom": "17.0.14",
+        "@types/react-tabs": "2.3.4",
+        "@types/styled-components": "5.1.26",
+        "@typescript-eslint/eslint-plugin": "5.11.0",
+        "auto-changelog": "2.4.0",
+        "autoprefixer": "10.4.14",
+        "babel-loader": "8.2.3",
+        "eslint": "8.40.0",
+        "eslint-config-impact-market": "impactMarket/eslint-config-impact-market",
```
Comment (obernardovieira) [inline package.json:53]: you can bump this to the actual release, 2.2.0

**Review by obernardovieira — CHANGES_REQUESTED** [review]
I'd like to get rid of `: any` as much as we can.

File: package.json (hunk 7 above)
Comment (hugomarquesdev) [reply package.json:53]: what do you mean? change the UI version in package.json from 1.5.6 to 2.2.0? (and bump it also as 2.2.0)

Comment (obernardovieira) [reply package.json:53]: no, the `eslint-config-impact-market`

**Review by obernardovieira — APPROVED** [review]

**Comment by obernardovieira** [comment]
@PauloSousapt

**Review by PauloSousapt — APPROVED** [review]

**Pull request merged**